src/components/AppInfo.js

```diff
@@ -0,0 +1,4 @@
+
+export default {
```

Comment:
I'm not entirely sure how this is intended to work.

Reply:
this is a default export. You can then include this as a normal module like this: import AppInfo form 'path to AppInfo'.

This export only happens one time, so this object is only created one time, even if you import it multiple times in multiple places.

when you access it, and modify any property of it, you are always using the same object reference. Like a singleton. 

In the future, if you want to add anything to this that is defined in the app initialization, you can then use it as a dependency in any module of the project